docs/book/component-gallery/orchestrators/kubernetes.md

```diff
@@ -63,6 +63,35 @@ You can now run any ZenML pipeline using the Kubernetes orchestrator:
 python file_that_runs_a_zenml_pipeline.py
 ```
 
+For additional configuration of the Kubernetes orchestrator, you can pass
+`KubernetesOrchestratorSettings` which allows you to configure the following attributes:
+
+* `node_affinity`: NodeAffinity match expressions.
+* `tolerations`: Toleration settings for the pod.
+
+```python
+from zenml.integrations.kubernetes.flavors import KubernetesOrchestratorSettings
+
+kubernetes_settings = KubernetesOrchestratorSettings(
+    node_affinity=[{
+        "key": "node.kubernetes.io/name",
+        "operator": "In",
+        "values": ['my_powerful_node_group']
+        }],
+    tolerations=[{"key": "node.kubernetes.io/name",
+                  "operator": "Exists",
+                  "value": "training",
+                  "effect": "NoSchedule"
+      }])
```

Comment:
this isn't formatted correctly.

I'd suggest to copy it into some random file, run the format script, then copy it back.